packages/valory/skills/abstract_round_abci/tests/conftest.py

```diff
@@ -18,14 +18,35 @@
 # ------------------------------------------------------------------------------
 
 """Conftest module for io tests."""
+import logging
+import os
+import shutil
+from contextlib import suppress
+from pathlib import Path
+from typing import Dict, Generator
+
+import pytest
+from hypothesis import settings
+
+from packages.valory.skills.abstract_round_abci.io_.store import StoredJSONType
+
 
 # pylint: skip-file
 
-from typing import Dict
 
-import pytest
+CI = "CI"
+PACKAGE_DIR = Path(__file__).parent.parent
+settings.register_profile(CI, deadline=5000)
 
-from packages.valory.skills.abstract_round_abci.io_.store import StoredJSONType
+
+@pytest.fixture(scope="session", autouse=True)
+def load_hypothesis_profile() -> Generator:
+    """Fixture to load hypothesis CI settings."""
+    if os.getenv(CI):
+        settings.load_profile(CI)
+    profile = settings.get_profile(settings._current_profile)
+    logging.info(f"Using hypothesis profile from {__file__}:\n{profile}")
+    yield
```

Comment:
this might need to be scoped to the module, because we still get flakiness because of test timing on CI that indicates a different deadline.
```
2022-12-07T11:32:56.6414495Z Unreliable test timings! On an initial run, this test took 294.39ms, which exceeded the deadline of 200.00ms, but on a subsequent run it took 2.60 ms, which did not. If you expect this sort of variability in your test timings, consider turning deadlines off for this test by setting deadline=None.
```